FinSearchUnified/Bundle/SearchBundleFindologic/QueryBuilder/NewSearchQueryBuilder.php

```diff
@@ -25,4 +25,12 @@ public function addCategories(array $categories)
         $categoryPath = implode('_', $categories);
         $this->addParameter('cat', $categoryPath);
     }
+
+    protected function setDefaultParameters()
```

Comment:
Was this overseen last time or why was it added just now?

Reply:
Yes, it was overseen. The click test was failing without it since `query` is a required parameter.